### PR TITLE
[FIX] column "payment_order_type" does not exist during migration to 9.0

### PR DESCRIPTION
**Pull request by metaminux**
#398

### CODE REVIEW

**Comment by metaminux** [comment]
It seems to me that "payment_order_type" is only present in "account_banking_payment_export" module, so I added a test on its state before calling the "map_payment_type" method.

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Comment by pedrobaeza** [comment]
I will prefer to test if the column exists. You have the method in openupgradelib called `column_exists`

**Comment by pedrobaeza** [comment]
Any news about this?

**Comment by metaminux** [comment]
Yes, I wanted to try "column_exists" version on a true migration.

I'll change my last commit in a few days I think.

File: account_payment_order/migrations/9.0.1.0.0/post-migration.py [hunk 1]
```diff
@@ -24,7 +24,9 @@ def map_payment_type(cr):
 @openupgrade.migrate(use_env=True)
 def migrate(env, version):
     cr = env.cr
-    map_payment_type(cr)
+    if openupgrade.table_exists(cr, 'account_payment_order') \
+       and openupgrade.column_exists(cr, 'account_payment_order', 'payment_order_type'):
```
Comment (pedrobaeza) [inline account_payment_order/migrations/9.0.1.0.0/post-migration.py]: Incorrect indentation an line very long (>80 chars)

Comment (metaminux) [reply account_payment_order/migrations/9.0.1.0.0/post-migration.py]: @pedrobaeza Could you please point me to an OCA guideline on how to break line in too long if statement, cause PEP8 is not really strict on that point (see https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator )

Thx

Comment (pedrobaeza) [reply account_payment_order/migrations/9.0.1.0.0/post-migration.py]: Indeed here there's no strict guideline, except complying with that 2 rules (indentation on second line of the if should be double and not > 80 cols). My proposal in this case is:

```python
    table = 'account_payment_order'
    if (openupgrade.table_exists(cr, table) and
            openupgrade.column_exists(cr, table, 'payment_order_type')):
```

Comment (metaminux) [reply account_payment_order/migrations/9.0.1.0.0/post-migration.py]: OK, done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**